src/Common/src/CoreLib/System/Type.Helpers.cs

```diff
@@ -342,7 +342,7 @@ public virtual bool IsAssignableFrom(Type c)
             // For backward-compatibility, we need to special case for the types
             // whose UnderlyingSystemType are runtime implemented. 
             Type toType = this.UnderlyingSystemType;
-            if (toType.IsRuntimeImplemented())
+            if (toType != null && toType.IsRuntimeImplemented())
```

Comment:
Nit: This can be `toType?.IsRuntimeImplemented()`

Reply:
Hmm since the bool is a primitive, using ? means the expression is now bool? requiring this:
`toType?.IsRuntimeImplemented() == true`
which may or may not be more readable than
`toType != null && toType.IsRuntimeImplemented()`

Reply:
Ok. `toType != null && toType.IsRuntimeImplemented()` looks better then.